db/userAddresses.js

```diff
@@ -0,0 +1,46 @@
+const client = require("./client");
+
+async function addShippingAddress({
+    userId,
+    street,
+    city,
+    state,
+    zip
+}) {
+    try{
+        const { rows: [shippingAddress] } = await client.query(`
+            INSERT INTO shippingAddresses("userId", street, city, state, zip)
+            VALUES ($1, $2, $3, $4, $5)        
```

Comment:
I think we want "ON CONFLICT ("userId") DO NOTHING" under here to prevent the same user from having multiple shipping addresses.

Reply:
Done